### PR TITLE
CI: attempt to fix "push translations"

### PR DESCRIPTION
**Pull request by 6543**
https://drone.gitea.io/go-gitea/gitea/67531/1/4

-> https://github.com/appleboy/drone-git-push/blob/master/DOCS.md ?

### CODE REVIEW

**Comment by delvh** [comment]
> fatal: couldn't find remote ref refs/pull/22964/head

Interesting.

**Comment by yardenshoham** [comment]
BTW this branch is open on upstream

**Comment by yardenshoham** [comment]
Can you close this and open one from a fork?

File: .drone.yml [hunk 1]
```diff
@@ -619,10 +619,7 @@ steps:
       commit: true
       commit_message: "[skip ci] Updated translations via Crowdin"
       remote: "git@github.com:go-gitea/gitea.git"
-    environment:
```
Comment (appleboy) [inline .drone.yml:630]: Just replace `appleboy/drone-git-push` with `appleboy/drone-git-push:linux-amd64`

**Review by appleboy — CHANGES_REQUESTED** [review]

File: .drone.yml (hunk 1 above)
Comment (appleboy) [reply .drone.yml:630]: @6543 I will update the latest image with multiple platforms.

**Comment by appleboy** [comment]
@6543 Can you help to restart the build? I already fixed the issue. See https://hub.docker.com/layers/appleboy/drone-git-push/latest/images/sha256-66e30db1203e9b9fe6400d2471630c0c5cfe25dad8217ef48901cc55e21720f3?context=repo

**Comment by lunny** [comment]
I think this pull request could be closed now because the push translations has been resolved. @6543

**Pull request closed without merging**